pom.xml

```diff
@@ -138,6 +140,11 @@
         <artifactId>aot</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Is this needed? I don't think it's ever used as a dependency

Reply:
right, moved to `pluginManagement`